Project.toml

```diff
@@ -0,0 +1,12 @@
+name = "ForwardDiff"
+uuid = "f6369f11-7733-5829-9624-2563aa707210"
```

Comment:
@fredrikekre Re: https://github.com/JuliaDiff/ForwardDiff.jl/pull/334#discussion_r204172753

Thanks, I fixed `uuid` in this PR.